docs/_posts/2016-01-04-pouchdb-5.2.0-magical-mystery-pouch.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+
+title: PouchDB 5.2.0&#58; Magical mystery pouch
+author: Nolan Lawson
+
+---
+
+> "Rollup! Rollup for the mystery tour!"
+> - Lennon and McCartney
+
+Besides a slew of bugfixes and the new `revs_limit` feature, the big news about 5.2.0 is that PouchDB is now authored in ES6 modules rather than CommonJS. Using [Rollup](http://rollupjs.org/), it's transpiled into one big `index.js` for Node and an `index-browser.js` for Webpack/Browserify.
```

Comment:
could this be "with ES6 modules" using "in" makes it read as if PouchDB is written in ES6 now almost?